js-intermediate.js

```diff
@@ -29,31 +47,38 @@ var bicycle = {
 }
 
 // Log the type of bicycle:
-console.log()
+console.log(bicycle.type)
 
 // Log the bell:
-console.log()
+console.log(bicycle.gear[2])
 
 // Log the PSI:
-console.log()
+console.log(bicycle.wheels.specs[2])
 
 
 // 4. Write a function that takes in an array and returns an array in reverse order.
-
-var originalArray1 = [1, 2, 3, 4, 5, 6, 7]
+var originalArray1 = [1, 2, 3, 4, 5, 6, 7];
 // Expected output: [7, 6, 5, 4, 3, 2, 1],
-var originalArray2 = ["9", "1", "o", "h", "c", "e"]
+var originalArray2 = ["9", "1", "o", "h", "c", "e"];
 // Expected output: ["e", "c", "h", "o", "1", "9"]
 
-
-
+var rev = () => {
+	return originalArray1.reverse()
+}
+console.log(rev())
 
```

Comment:
You have the right idea here, but rather than reversing just one hard coded array, use an argument placeholder so you can pass in any array to the function.
var rev = (array) => {
     return array.reverse()
}
console.log(rev(originalArray1))
console.log(rev(originalArray2))